slam-ws/src/load_waypoints/scripts/load_waypoints_server.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import rospy, os, json 
+from load_waypoints.srv import *
+
+all_waypoints = list()
+
+def populate_waypoint_table():
+    base_dir = rospy.get_param('~arg_name') # ~ added to arg_name because private param 
+    
+    with open(base_dir + '/scripts/waypoints.json') as f:
+        try:
+            data = json.load(f)
+        except ValueError, e:
+            print("Invalid JSON")
+            sys.exit(1)
+
+    # Parse through dictionary and create list of lists holding all waypoints
+    for waypoint in data["waypoints"]:
+        all_waypoints.append([waypoint['coordinate 1'], waypoint['coordinate 2']])
+
+    print("Loaded waypoints:", all_waypoints)
```

Comment:
Forgot this before but, we should use `rospy.loginfo(...)` to log instead of `print`.